tests/characters/control/federated/test_rpc_control_f.py

```diff
@@ -8,21 +8,22 @@
 
 
 def validate_json_rpc_response_data(response, method_name, specification):
-    _input_fields, required_output_fileds = specification.get_specifications(interface_name=method_name)
+    _input_fields, _optional, required_output_fileds = specification.get_specifications(interface_name=method_name)
     assert 'jsonrpc' in response.data
     for output_field in required_output_fileds:
         assert output_field in response.content
     return True
 
 
 def test_alice_rpc_character_control_create_policy(alice_rpc_test_client, create_policy_control_request):
+    alice_rpc_test_client.__class__.MESSAGE_ID = 0
     method_name, params = create_policy_control_request
     request_data = {'method': method_name, 'params': params}
     rpc_response = alice_rpc_test_client.send(request=request_data)
     assert rpc_response.success is True
     assert rpc_response.id == 1
 
-    _input_fields, required_output_fileds = alice_specification.get_specifications(interface_name=method_name)
+    _input_fields, _optional, required_output_fileds = alice_specification.get_specifications(interface_name=method_name)
```

Comment:
I wonder if this API wants to be access-by-key instead of unpacking now.

Reply:
Yes, that can be done. Do you think it's strictly better?

Reply:
No - not strictly better - In fact I think the *primary* use case is to unpack 'em all. So I think I like it as-is.